standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -1308,27 +1308,43 @@ struct WMCreateOrDropTriggerToPoolMappingResponse {
 // Schema objects
 // Schema is already taken, so for the moment I'm calling it an ISchema for Independent Schema
 struct ISchema {
-  1: SchemaType schemaType,
+  1:  optional i64 schemaId,
```

Comment:
Since there's been a release you won't be able to renumber the thrift struct components.  You'll have to put schemaId at the end.